types/fusebit__add-on-sdk/fusebit__add-on-sdk-tests.ts

```diff
@@ -0,0 +1,77 @@
+import Sdk = require('fusebit__add-on-sdk');
```

Comment:
```suggestion
import Sdk = require('@fusebit/add-on-sdk');
```

Reply:
This suggestion results in:
```
Error: /hp/DefinitelyTyped/types/fusebit__add-on-sdk/fusebit__add-on-sdk-tests.ts:1:22
ERROR: 1:22   expect  TypeScript@4.9 compile error: 
Cannot find module '@fusebit/add-on-sdk' or its corresponding type declarations.
ERROR: 67:67  expect  TypeScript@4.9 compile error: 
Parameter 'client' implicitly has an 'any' type.
ERROR: 75:73  expect  TypeScript@4.9 compile error: 
Parameter 'url' implicitly has an 'any' type.
```

Reply:
This change results in:
```
Error: /hp/DefinitelyTyped/types/fusebit__add-on-sdk/fusebit__add-on-sdk-tests.ts:1:22
ERROR: 1:22   expect  TypeScript@4.9 compile error: 
Cannot find module '@fusebit/add-on-sdk' or its corresponding type declarations.
ERROR: 67:67  expect  TypeScript@4.9 compile error: 
Parameter 'client' implicitly has an 'any' type.
ERROR: 75:73  expect  TypeScript@4.9 compile error: 
Parameter 'url' implicitly has an 'any' type.
```

Reply:
This change results in:
```
Error: /hp/DefinitelyTyped/types/fusebit__add-on-sdk/fusebit__add-on-sdk-tests.ts:1:22
ERROR: 1:22   expect  TypeScript@4.9 compile error: 
Cannot find module '@fusebit/add-on-sdk' or its corresponding type declarations.
ERROR: 67:67  expect  TypeScript@4.9 compile error: 
Parameter 'client' implicitly has an 'any' type.
ERROR: 75:73  expect  TypeScript@4.9 compile error: 
Parameter 'url' implicitly has an 'any' type.
```

Reply:
You need to add a `paths` entry to the tsconfig.json for it to work. This is the preferred way, but there are several packages that fail to do this already and it doesn’t matter a ton since this is in tests, so I won’t block on it, but if you want to fix it that would be great. You can look at `types/xmpp__client/tsconfig.json` for an example.

Reply:
Fixed.